src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/androids/ProgrammableAndroid.java

```diff
@@ -706,10 +706,9 @@ private void executeInstruction(Instruction instruction, Block b, BlockMenu inv,
                     BlockStorage.addBlockInfo(b, "index", String.valueOf(0));
                     break;
                 case CHOP_TREE:
-                    // We only move to the next step if we finished chopping wood
-                    if (chopTree(b, inv, face)) {
-                        BlockStorage.addBlockInfo(b, "index", String.valueOf(index));
-                    }
+                    // Move to the next step and try to chop the tree
+                    BlockStorage.addBlockInfo(b, "index", String.valueOf(index));
+                    chopTree(b, inv, face);
```

Comment:
This will make the android only mine a single block...

Reply:
It previously only called it once

Reply:
You are advancing the `index` on every iteration now.
Previously the `index` was only incremented if the woodcutting operation has finished.